src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+
+function convertToNumber(currency) {
+  const str = currency.slice(1);
+  let result = '';
+
+  for (let i = 0; i < str.length; i++) {
+    if (str[i] === ',') {
+      result += '.';
+    } else {
+      result += str[i];
+    }
+  }
+
+  return +result;
+}
```

Comment:
The `convertToNumber` function could be simplified. Instead of manually replacing commas with dots and then converting the string to a number, you could use a regular expression to remove all non-digit characters, then use the `Number` function to convert the resulting string to a number.